docs/hello.rst

```diff
@@ -171,9 +188,23 @@ In the fractal code above, ``for i, j in pixels`` loops over all the pixel coord
           for i in x:
             ...
 
-.. warning::
+.. note::
+    ``break`` is not supported in **outermost (parallelized)** loops:
 
-    Struct-for's must be at the outer-most scope of kernels.
```

Comment:
Shall we keep this line?

Reply:
I thought this is (and can be) supported?
```
import taichi as ti

x = ti.var(ti.i32, 3)
y = ti.var(ti.i32, 3)

@ti.kernel
def func():
    for i in x:
        for j in y:
            x[i] *= y[j]

func()
```

Reply:
No, this is not supported.